docs/privacy-policy.md

```diff
@@ -29,7 +29,7 @@ None. We don’t collect your Personal Data, period.
 We do not collect, rent, store or sell your Personal Data to anyone. However
 because IPFS Companion is a web extension that provides access to the real-time,
 peer-to-peer IPFS Network (which is a public platform for which anyone may join
-and participate) the data that you add or upload to the IPFS Network using IPFS
```

Comment:
Wanted to call this out because `upload` in the context of the privacy policy may have implications I don't know about, and didn't want it to be merged silently.

Reply:
I really appreciate this, thank you!

Two asks:
- In this case it makes more sense to remove `add` and just keep `import` (it covers more ground)
- Also, there is a date labeled `Last Update` at the top of the file, please bump it to current date (helps when we copy & paste it to addon stores)